rn/Teacher/ios/en-AU.lproj/Localizable.stringsdict

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>grades_currently_hidden</key>
+    <dict>
+        <key>NSStringLocalizedFormatKey</key>
+        <string>%#@VARIABLE@</string>
+        <key>VARIABLE</key>
+        <dict>
+            <key>NSStringFormatSpecTypeKey</key>
+            <string>NSStringPluralRuleType</string>
+            <key>NSStringFormatValueTypeKey</key>
+            <string>d</string>
+            <key>zero</key>
+            <string>%d grades currently hidden</string>
+            <key>one</key>
+            <string>%d grade currently hidden</string>
+            <key>other</key>
+            <string>%d grades currently hidden</string>
+        </dict>
+    </dict>
+    <key>grades_currently_posted</key>
+    <dict>
+        <key>NSStringLocalizedFormatKey</key>
+        <string>%#@VARIABLE@</string>
+        <key>VARIABLE</key>
+        <dict>
+            <key>NSStringFormatSpecTypeKey</key>
+            <string>NSStringPluralRuleType</string>
+            <key>NSStringFormatValueTypeKey</key>
+            <string>d</string>
+            <key>zero</key>
+            <string>%d grades currently posted</string>
+            <key>one</key>
+            <string>%d grade currently posted</string>
+            <key>other</key>
+            <string>%d grades currently posted</string>
+        </dict>
+    </dict>
+</dict>
+</plist>
```

Comment:
I just copied the English to en-AU so that it would add the disclosure triangle for locale variants of the file. This probably wasn't necessary. Moving the base file into the en.lproj by hitting the "Localize" button is probably enough to start asking for translations for this.